cla-backend/cla/models/dynamo_models.py

```diff
@@ -975,6 +977,20 @@ def __init__(self, user_email=None, user_external_id=None, user_github_id=None,u
         self.model.lf_username = lf_username
         self.model.lf_sub = lf_sub
 
+    def __str__(self):
+        return 'id:{}, username: {}, gh id:{}, gh username:{}, ' + \
```

Comment:
I wouldn't use `+' concatenation in conjunction with a .format here. Instead I would add parentheses. e.g.

```
return ('this is a really long string'
        'that uses parentheses. User: {}').format(user)
```